tools/clang/test/HLSLFileCheck/hlsl/types/vector/vec_size_annot.hlsl

```diff
@@ -0,0 +1,20 @@
+// RUN: %dxc -T ps_6_8 -E main -Qkeep_reflect_in_dxil -select-validator internal %s | FileCheck %s
```

Comment:
This might highlight a potential deficiency in our lit shell test path in comparison to our TAEF FileCheck path.  In the TAEF FileCheck path, this is automatically inferred from the -T argument on the dxc command line.  I don't think we have the equivalent in the lit shell test path.

Reply:
BTW: this test has -select-validator internal, so it doesn't technically need any `%dxilver` check.  It's intended to skip external validation, and embed the reflection into the main module to allow metadata check in the test.  Otherwise, that metadata would be stripped from the main module, and only exist in the reflection module which isn't disassembled.